release-notes/3.0/preview/api-diff/preview1/3.0-preview1_System.Runtime.Intrinsics.X86.md

```diff
@@ -0,0 +1,673 @@
+# System.Runtime.Intrinsics.X86
+
+``` diff
+ namespace System.Runtime.Intrinsics.X86 {
```

Comment:
Not sure if it's worth mentioning, but for the HWIntrinsics, the snap happened in a somewhat irregular place and there are several APIs (basically the delta between preview1 and preview2) that no longer exist and will always throw PNSE when called (if on preview1).